src/lib/CookieJar.ts

```diff
@@ -43,6 +44,11 @@ export default class CookieJar {
             const walletIdChunks = wallet.id.match(/.{2}/g);
             for (const chunk of walletIdChunks!) bytes.push(parseInt(chunk, 16));
 
+            // status
+            let statusByte: number = 0;
+            statusByte = statusByte | (wallet.keyMissing ? 1 : 0);
```

Comment:
I suggest using `0b1` and `0b0`.

Reply:
In this case it would be kind of alright to use `0b0`, but looking at the 7th bit it is just unnecessarily complex i think.

```suggestion
statusByte = statusByte | (wallet.keyMissing ? CookieJar.StatusFlags.KEY_MISSING : 0b0000000);
```
which in the end does exactly the same thing, but I think is even less understandable.